osCam/userconfig/forms.py

```diff
@@ -27,14 +27,20 @@ class Meta:
         model = Camera 
         fields = ('model_info',)
 
+# alternate carera form whoops
+# class CameraForm(forms.ModelForm):
+#     class Meta():
+#         model = Camera
+#         fields = ('recording', 'fps', 'invert', 'mirror', 'codec', 'camera_index', 'device_name', 'ip_address', 'port', 'username', 'password')
 
-class RaspberryPiEntryForm( ModelForm ):
-    model_num = forms.CharField(label='model number',
-        widget=forms.TextInput(attrs={'size':'80', 'class':"form-control"})
-        )
-    model_name = models.CharField(label='model name',
-        widget=forms.TextInput(attrs={'size':'80', 'class':"form-control"})
-        )
-    class Meta:
-        model = RaspberryPi 
-        fields = ('model_num', 'model_name')
+
+# class RaspberryPiEntryForm( ModelForm ):
+#     model_num = forms.CharField(label='model number',
+#         widget=forms.TextInput(attrs={'size':'80', 'class':"form-control"})
+#         )
+#     model_name = models.CharField(label='model name',
+#         widget=forms.TextInput(attrs={'size':'80', 'class':"form-control"})
+#         )
+#     class Meta:
+#         model = RaspberryPi 
+#         fields = ('model_num', 'model_name')
```

Comment:
may I ask why?

Reply:
was giving me grief lol misuse of something or other. wasn't what i was working on so just commented it out. 

all these convos making me think i (and others) could benefit from having a quick-fix branch and a ready-for-PR branch. 

a lot of these are just things that were stopping progress but were not what i was working on so just killed them. 
this one above was an issue with how labels were being assigned. couldn't run the server without commenting it out.